modules/Bio/EnsEMBL/Transcript.pm

```diff
@@ -3142,7 +3142,18 @@ sub summary_as_hash {
   $summary_ref->{'havana_version'} = $havana_transcript->version() if defined $havana_transcript;
   $summary_ref->{'ccdsid'} = $self->ccds->display_id() if $self->ccds();
   $summary_ref->{'transcript_support_level'} = $self->tsl() if $self->tsl();
-  $summary_ref->{'tag'} = 'basic' if $self->gencode_basic();
+
+  my @tags;
+  push @tags, 'basic' if $self->gencode_basic();
+  push @tags, 'Ensembl_canonical' if $self->is_canonical();
+  push @tags, 'primary' if $self->gencode_primary();
+
+  my $mane = $self->mane_transcript();
+  if ($mane) {
+    push @tags, $mane->type() if ($mane->type());
```

Comment:
@sgiorgetti Do we need to define `type` for this `mane_transcript` object? I can see in the REST server that `type` is available and printed out but I wasn't sure where it's being defined, and if it needs to be defined here?

Reply:
@nwillhoft thanks for bringing this up. Reporting here the result of our discussion for the records.
The `mane_transcript` Transcript method returns a `MANE` object, which has the `type` method defined in it. This latter method returns the "type" of MANE - i.e. "MANE_Select" or "MANE_Plus_Clinical".